plugins/by-name/avante/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib, pkgs, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts mkNullOrOption' toLuaObject;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "avante";
+  originalName = "avante.nvim";
+  package = "avante-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  # TODO: remove when XXX reaches `nixos-unstable`
```

Comment:
do you have an associate override update PR for this?

Reply:
I am still looking for it...

Reply:
Actually I was wrong. Everything is find and we shouldn't have to add the dependencies there.